crates/router/src/compatibility/stripe/customers.rs

```diff
@@ -110,8 +110,9 @@ pub async fn customer_update(
     };
 
     let customer_id = path.into_inner();
-    let mut cust_update_req: customer_types::CustomerRequest = payload.into();
+    let mut cust_update_req: customer_types::CustomerUpdateRequest = payload.into();
     cust_update_req.customer_id = Some(customer_id);
+    let customer_update_id = customer_types::UpdateCustomerId::new("temp_global_id".to_string());
```

Comment:
Here to mention in code

Reply:
compatibility layer is going to deprecated in v2 right